CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Please take a moment to review this document before submitting a pull request.
+
+## Coding standards
+
+Our code formatting rules are defined in .prettierrc. You can automatically format your code against these standards by running:
+
+```bash
+yarn format
+```
+
+## Running tests
```

Comment:
I moved the testing stuff from the readme to here, and split it into two sections: running and writing. I also decided to keep the `yarn test` command in the readme since that's more of a high-level thing.